src/arrayReverse.test.js

```diff
@@ -8,13 +8,50 @@ describe(`Function 'arrayReverse':`, () => {
   });
 
   it(`should return an array`, () => {
-
+    expect(Array.isArray(arrayReverse(['first', 'array']))).toBe(true);
   });
 
   it(`should return an empty string
-    if original array consists of an empty string`, () => {
+   if original array consists of an empty string`, () => {
+    expect(arrayReverse([])).toEqual([]);
+  });
+
+  it(`should handle an array with an onechar word`, () => {
+    expect(arrayReverse(['h'])).toEqual(['h']);
+  });
+
+  it(`should handle an array with a single word of chars`, () => {
+    expect(arrayReverse(['hello'])).toEqual(['olleh']);
+  });
+
+  it(`should handle an array with a single word of numbers`, () => {
+    expect(arrayReverse(['123'])).toEqual(['321']);
+  });
+
+  it(`should handle an array with a single word of special symbols`, () => {
+    expect(arrayReverse(['+-!'])).toEqual(['!-+']);
+  });
+
+  it(`should handle an array with a single word of chars and numbers`, () => {
+    expect(arrayReverse(['hell0'])).toEqual(['0lleh']);
+  });
+
+  it(`should handle an array with a single word of chars and special 
+   chars`, () => {
+    expect(arrayReverse(['hello!'])).toEqual(['!olleh']);
+  });
```

Comment:
These tests can be combined in 1

Reply:
I mixed all tests at one test `should handle an array with a single word`, but I left the test `should handle an array with one symbol`